### PR TITLE
Optimize logic of DubboBeanDefinitionParser.java

### PR DESCRIPTION
**Pull request by tswstarplanet**
## What is the purpose of the change

Make code of process callback of method cleaner in DubboBeanDefinitionParser

## Brief changelog

DubboBeanDefinitionParser.java

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/schema/DubboBeanDefinitionParser.java [hunk 1]
```diff
@@ -389,9 +381,10 @@ private static void parseArguments(String id, NodeList nodeList, RootBeanDefinit
         }
     }
 
+
```
Comment (lixiaojiee) [inline dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/schema/DubboBeanDefinitionParser.java]: It looks like there are two more lines here.

File: dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/schema/DubboBeanDefinitionParser.java [hunk 2]
```diff
@@ -190,24 +194,12 @@ private static BeanDefinition parse(Element element, ParserContext parserContext
...
-                                } else if ("oninvoke".equals(property)) {
-                                    int index = value.lastIndexOf(".");
-                                    String invokeRef = value.substring(0, index);
-                                    String invokeRefMethod = value.substring(index + 1);
-                                    reference = new RuntimeBeanReference(invokeRef);
-                                    beanDefinition.getPropertyValues().addPropertyValue("oninvokeMethod", invokeRefMethod);
+                                    String ref = value.substring(0, index);
+                                    String method = value.substring(index + 1);
+                                    reference = new RuntimeBeanReference(ref);
+                                    beanDefinition.getPropertyValues().addPropertyValue(property + METHOD, method);
```
Comment (complone) [inline dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/schema/DubboBeanDefinitionParser.java]: Remove extra spaces

**Comment by tswstarplanet** [comment]
there is one problem, i will commit again

**Comment by tswstarplanet** [comment]
> there is one problem, i will commit again

It's ok

File: dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/schema/DubboBeanDefinitionParser.java (hunk 2 above)
Comment (tswstarplanet) [reply dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/schema/DubboBeanDefinitionParser.java]: I think there are no extra spaces

**Pull request merged**